homeassistant/components/unifiprotect/services.yaml

```diff
@@ -0,0 +1,33 @@
+set_doorbell_message:
+  name: Set Doorbell message
+  description: >
+    Use to dynamically set the message on a Doorbell LCD screen. Should only be used to set dynamic messages
+    (i.e. setting the current outdoor temperature on your Doorbell). Static messages should still using the Select entity and the
+    add_doorbell_text / remove_doorbell_text services.
+  fields:
```

Comment:
We can use

````
  target:
    entity:
      integration: unifiprotect
```

Add `domain: camera` if it should be restricted from the docs

Reply:
I am not sure I follow. It is already restricted to the domain select. The service is bound to the `select.unifiprotect` platform.